Lpc compute docs (#2651)
buildkite/docs: pages/buildkite_compute/pipeline_migration.md

> ## Compute pipeline migration

Reviewer: ```suggestion
## Hosted agents pipeline migration
```

> To migrate an existing pipeline to use a compute queue, you must first ensure:

Reviewer: ```suggestion
To migrate an existing pipeline to use a hosted agents queue, you must first ensure:
```

> - Your pipeline is in the same cluster as all the compute queues you wish to target.

Reviewer: ```suggestion
- Your pipeline is in the same cluster as all the hosted agent queues you wish to target.
```

> - Each step in the pipeline targets the required compute queue.

Reviewer: ```suggestion
- Each step in the pipeline targets the required hosted agent queue.
```

- Source control settings have been updated to allow code access.

> There are different pipeline repository settings required for public and private repositories, see below for the relevant instructions.

Author: change to "There are different repository settings required for public and private...

> There are different pipeline repository settings required for public and private repositories, see below for the relevant instructions.

Reviewer: Remove 'repository' from 'pipeline repository settings' since this setting relates to 'pipeline settings'.

```suggestion
Different pipeline settings are required for public and private repositories, see below for the relevant instructions.
```

### Private repository

> Before completing the following steps ensure you have followed the instructions in [Private Repository Access](/docs/buildkite-compute/source-control#compute-code-access-private-repositories).

Reviewer: More active voice...

```suggestion
Ensure you have followed the instructions in [Private Repository Access](/docs/buildkite-compute/source-control#compute-code-access-private-repositories) before completing the following steps.
```

- Set your pipeline to use the GitHub (with code access) service you authorized in the step above.

> - Navigate to your pipeline settings.

Author: Add additional level of indentation here

- Select GitHub from the left menu.  
- Remove the existing repository, or select the _Choose another repository or URL_ link
- Select the GitHub account including ...(with code access).
- Select the repository.
- Select _Save Repository_.

## Public repository

> When accessing a public repository from a Buildkite compute agent, you must ensure the services is using `HTTPS` for checkout.

Reviewer: ```suggestion
When accessing a public repository from a Buildkite hosted agent, you must ensure the services is using `HTTPS` for checkout.
```
